tests/store_helpers.py

```diff
@@ -73,12 +73,33 @@ def ensure_application_version(
             store.add_commit(version)
         return version
 
+    def ensure_application(
```

Comment:
Will we still need the `add_application()` method if this exists?

Reply:
Probably, it is used in many places.